integrations/tensorflow/e2e/math_test.py

```diff
@@ -29,6 +29,10 @@ def abs(self, x):
   def ceil(self, x):
     return tf.math.ceil(x)
 
+  @tf.function(input_signature=[tf.TensorSpec([4], tf.float32)])
+  def compare(self, x):
```

Comment:
something like `greater_than` might be more descriptive here, and give the logs this creates more context.